app/controllers/accounts_controller.rb

```diff
@@ -4,7 +4,9 @@ class AccountsController < ApplicationController
   PAGE_SIZE = 20
 
   include AccountControllerConcern
+  include RoutingHelper
```

Comment:
Can we change the view for this (I'm assuming this is the problem) so that it uses the `TagManager`s and we don't need to include `RoutingHelper`?

Reply:
yeah, we can do that.

Reply:
(i think it might have been the controllers that inherit from it that use the helpers)